cpp/ql/src/Metrics/Internal/DiagnosticsSumElapsedTimes.ql

```diff
@@ -0,0 +1,9 @@
+/**
+ * @name Sum of frontend and extractor time
+ * @description The sum of elapsed frontend time, and the sum of elapsed extractor time.
+ * @kind table
+ * @id cpp/frontend-and-extractor-time
+ */
+
+select sum(@compilation c, float seconds | compilation_time(c, _, 2, seconds) | seconds) as sum_frontend_elapsed_seconds,
+  sum(@compilation c, float seconds | compilation_time(c, _, 4, seconds) | seconds) as sum_extractor_elapsed_seconds
```

Comment:
If there's a reason why you can't `import cpp` and use the `Compilation` class, then please write that as a code comment.

Reply:
Good point! Maybe I can 🤔 I just copied this query from the internal directory without giving it much thought, really.